Checklist: Rework the checklist according to the current situation (#27)
WebEngDUS/WebEngDUS: CHECKLISTS.md

# Checklists for running a Web Engineering DUS Meetup

## Before the meetup: Planning phase

- [ ] Think about what format we want to run for the next meetup (one talk, multiple talks, a workshop, etc.)
- [ ] Align on a day with your Co-Organizer when both have time for a meetup
- [ ] Search a Meetup-Location by contacting a few local companies

> - [ ] Once the date is confirmed with the Location sponsor, search for one or more speaker

Reviewer: ```suggestion
- [ ] Once the date is confirmed with the location sponsor, search for one or more speakers
```

> - [ ] Once the date is confirmed with the Location sponsor, send over the Location briefing (see [README.md](./README.md))

Reviewer: ```suggestion
- [ ] Once the date is confirmed with the location sponsor, send over the location briefing (see [README.md](./README.md))
```

> - [ ] Once the date is confirmed with the Location sponsor, block the aligned date and time in your private calendars

Reviewer: ```suggestion
- [ ] Once the date is confirmed with the location sponsor, block the aligned date and time in your private calendars
```

> - [ ] Contact the speaker and try to align a date

Reviewer: ```suggestion
- [ ] Contact the speaker(s) and try to align a date
```

> - [ ] Once the speaker is confirmed, send over the Speaker briefing (see [README.md](./README.md))

Reviewer: ```suggestion
- [ ] Once the speakers are confirmed, send over the speaker briefing (see [README.md](./README.md))
```

> - [ ] Setup the appointment on [Meetup.com](https://www.meetup.com/), send it to the speaker, and announce it

Reviewer: ```suggestion
- [ ] Set up the appointment on [Meetup.com](https://www.meetup.com/), send it to the speakers, the location sponsor and announce it publicy
```
